v2/apiserver/schemas/event.json

```diff
@@ -62,6 +62,9 @@
 				},
 				"commit": {
 					"type": "string",
+					"pattern": "^[a-f0-9]*$",
```

Comment:
Are git shas _always_ lowercase?

I sincerely do not know the answer to this.

Reply:
All signs point to yes -- however, I just played around with taking a legit commit sha, updating many of the alphabetical characters to uppercase, and git had no qualms checking out the commit.  With that in mind, I'd be comfortable allowing `A-F` as well.  What do you think?

Reply:
By Postel's principle... yes... let's accept both.

Reply:
Updated.